backend/schema/validate.go

```diff
@@ -169,9 +169,31 @@ func ValidateModuleInSchema(schema *Schema, m optional.Option[*Module]) (*Schema
 				}
 				return next()
 
+			case *MetadataCalls:
```

Comment:
Nice, but I don't see checks in this file for type references? I'd expect a `Ref` case in here to be checking if the target data structure/enum/etc. is exported?

Reply:
Oh I like that! I'll give it a go!